patsicko/my-brand-bakend
# Ft blog create 005

## What this PR do
Create a blog
## Description of the tasks to be completed
POST /api/createBlog
[Finishes#005]

## How should this be manually tested
- open postman
- set your HTTP request to POST
- input API endpoint
- switch to the Body tab
- Click raw
- select JSON
-  create a json object with blogTitle, blogAutor,blogImane and blogText for test
- Click Send.
- Status: 201 Created should be displayed
- Posted data are showing up in the body.

## Trello story id
#003
